addons/knowledge/controllers/main.py

```diff
@@ -222,6 +222,7 @@ def get_article_permission_panel_data(self, article_id):
                 'based_on': f'{member["based_on_icon"] or "📄"} {member["based_on_name"]}' if member['based_on_name'] else False,
```

Comment:
Some points in your commit message concern the previous commit (reloading the tree...)